docs/src/examples/components/Segment/Variations/SegmentExampleColor.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Segment, ProviderConsumer } from '@stardust-ui/react'
+
+const SegmentExampleColor = () => (
+  <ProviderConsumer
+    render={({ siteVariables: { emphasisColors, naturalColors } }) =>
```

Comment:
are `contextualColors` supported as well as a value for `color` prop? (not saying that those should be included in the example, but would like to clarify that)

Reply:
apparently, no - only emphasis and natural ones, according to the way `mapColorsToScheme` is implemented. Could you, please, suggest what is the reason behind that? As a client, I would expect that the following should be possible as well:

```jsx
<Segment color="danger">Do you really want to remove the repo?</Segment>
```

but this won't work.

Reply:
@kuzhelov https://github.com/stardust-ui/react/pull/597#discussion_r241343539 should clarify that partially, we still do not support `danger` prop; can we clarify this? @layershifter @levithomason

Reply:
lets leave it for a discussion, will make a note about that